packages/@aws-cdk/aws-scheduler-targets-alpha/lib/event-bridge-put-events.ts

```diff
@@ -0,0 +1,107 @@
+import { IScheduleTarget, ISchedule, ScheduleTargetInput, ScheduleTargetConfig } from '@aws-cdk/aws-scheduler-alpha';
+import { Names } from 'aws-cdk-lib';
+import * as events from 'aws-cdk-lib/aws-events';
+import { IRole } from 'aws-cdk-lib/aws-iam';
+import * as iam from 'aws-cdk-lib/aws-iam';
+import { ScheduleTargetBase, ScheduleTargetBaseProps } from './target';
+import { sameEnvDimension } from './util';
+
+/**
+ * An entry to be sent to EventBridge
+ *
+ * @see https://docs.aws.amazon.com/eventbridge/latest/APIReference/API_PutEventsRequestEntry.html
+ */
+export interface EventBridgePutEventsEntry {
+  /**
+   * The event body
+   *
+   * Can either be provided as an object or as a JSON-serialized string
+   * @example
+   *
+   * ScheduleTargetInput.fromText('{"instance-id": "i-1234567890abcdef0", "state": "terminated"}');
+   * ScheduleTargetInput.fromObject({ Message: 'Hello from a friendly event :)' });
+   */
+  readonly detail: ScheduleTargetInput;
+
+  /**
+   * Used along with the source field to help identify the fields and values expected in the detail field
+   *
+   * For example, events by CloudTrail have detail type "AWS API Call via CloudTrail"
+   * @see https://docs.aws.amazon.com/eventbridge/latest/userguide/eb-events.html
+   */
+  readonly detailType: string;
+
+  /**
+   * The event bus the entry will be sent to.
+   *
+   */
+  readonly eventBus: events.IEventBus;
+
+  /**
+   * The service or application that caused this event to be generated
+   *
+   * Example value: `com.example.service`
+   *
+   * @see https://docs.aws.amazon.com/eventbridge/latest/userguide/eb-events.html
+   */
+  readonly source: string;
+}
+
+/**
+ * Send an event to an AWS EventBridge by AWS EventBridge Scheduler.
+ */
+export class EventBridgePutEvents extends ScheduleTargetBase implements IScheduleTarget {
+
+  constructor(
+    private readonly entry: EventBridgePutEventsEntry,
+    private readonly props: ScheduleTargetBaseProps,
+  ) {
+    super(props, entry.eventBus.eventBusArn);
+    if (this.baseProps.input) {
+      throw new Error('ScheduleTargetBaseProps.input is not supported for EventBridgePutEvents. Please use entry.detail instead.');
+    }
```

Comment:
I don't quite understand this even with your pr description. `detail` and `input` seem to both have the same type. Why do we accept one but not the other?

Reply:
Theoretically we could accept both (`input` and `detail`). However if somebody is supplying both, which should we pick? 
Then you might think that we can make both optional and throw an exception if both are supplied, but I dont see any advantages going that route.

We could also omit `detail` from `EventBridgePutEventsEntry`, but I think that might be confusing for users, who know the structure from the [PutEvents API Call](https://docs.aws.amazon.com/eventbridge/latest/APIReference/API_PutEvents.html), eg. `detail`. And I think its a bit unintuitive to put most of the payload in one object and the main part (e.g detail in input). So this was my reasoning for the current implementation.

Feel free to question me further or share me how you think about this. :)
I hope I was able to describe it good enough to understand.